src/pip/_vendor/README.rst

```diff
@@ -16,7 +16,7 @@ Vendoring Policy
   pure Python.
 * Any modifications made to libraries **MUST** be noted in
   ``pip/_vendor/README.rst`` and their corresponding patches **MUST** be
-  included ``tools/automation/vendoring/patches``.
+  included ``tools/vendoring/patches``.
```

Comment:
These README changes are kind of unrelated, but there seems to be an error here and I was too lazy to create a separate PR. :smile: If `pip` is very strict about "PR does one thing only", I'll revert ofc.